i18n/ru/docusaurus-plugin-content-docs/current/reference/isolation/decouple-entities.mdx

```diff
@@ -9,7 +9,7 @@ import WIP from '@site/src/shared/ui/wip/tmpl.mdx'
 
 <WIP ticket="202" />
 
-> Про кросс-импорты типов, адаптеро и про то - как явно выстраивать связи между сущностями
+> Про кросс-импорты типов, адаптера и про то - как явно выстраивать связи между сущностями
```

Comment:
suggestion: I think it will sound better like this:
```suggestion
> Про кросс-импорты типов, адаптеры и про то - как явно выстраивать связи между сущностями
```

Reply:
Yeah. Sounds good.